cirq-core/cirq/ops/pauli_string.py

```diff
@@ -984,7 +984,9 @@ def conjugated_by(self, clifford: 'cirq.OP_TREE') -> 'PauliString':
         for op in list(op_tree.flatten_to_ops(clifford))[::-1]:
             if pauli_map.keys().isdisjoint(set(op.qubits)):
                 continue
-            for clifford_op in _decompose_into_cliffords(op)[::-1]:
```

Comment:
please find every use of `_decompose_into_cliffords ` in this file and replace it with the construction a CliffordGate ... and then delete that method